### PR TITLE
docs/reference: update doc for PD Region Meta-Storage

### PR DESCRIPTION
**Pull request by dcalvin**
## What have you changed? (mandatory)

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
- Doc update. Related Jira ticket: [TIKV-3583](https://internal.pingcap.net/jira/browse/TIKV-3583)

## What are the type of the changes? (mandatory)
- Doc update

## Does this PR affect documentation (docs) or release note? (mandatory)

- documentation .

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @nolouch

File: docs/reference/tools/pd-control.md [hunk 1]
```diff
@@ -98,48 +98,119 @@ Use this command to view or modify the configuration information.
...
-  "disable-make-up-replica": "false",
-  "disable-remove-extra-replica": "false",
-  "disable-location-replacement": "false",
-  "disable-namespace-relocation": "false",
-  "schedulers-v2": [
-    {
-      "type": "balance-region",
-      "args": null,
-      "disable": false
+  "client-urls": "http://127.0.0.1:2379",
```
Comment (Hoverbear) [inline docs/reference/tools/pd-control.md]: Maybe we can just give a few samples then say `// ... Omitted`? It seems too long and our config changes with new versions anyways, saves us updating it each time.

File: docs/reference/tools/pd-control.md [hunk 2]
```diff
@@ -274,6 +345,12 @@ The configuration above is global. You can also tune the configuration by config
 
 - `disable-namespace-relocation` is used to disable Region relocation to the store of its namespace. When you set it to `true`, PD does not move Regions to stores where they belong to.
 
+- `use-region-storage` is used to enable or disable Region metadata storage in PD.  When you set it to `true`, the metadata of PD Regions will be saved to the Region Meta-Storage, a seperate storage engine. This solves the potential performance issue with BlotDB (backend of etcd) that may occur if the stored metadata has reached a GB level. Metadata is synchonized across multiple PD servers and eventally consistency is guranteed through Raft. The default value is `true`.
```
Comment (Hoverbear) [inline docs/reference/tools/pd-control.md]:
```suggestion
- `use-region-storage` is used to enable or disable Region metadata storage in PD.  When you set it to `true`, the metadata of PD Regions will be saved to the Region Meta-Storage, a separate storage engine. This solves the potential performance issue with BlotDB (backend of etcd) that may occur if the stored metadata has reached a GB level. Metadata is synchronized across multiple PD servers and eventually consistency is guaranteed through Raft. The default value is `true`.
```

File: docs/reference/tools/pd-control.md (hunk 1 above)
Comment (dcalvin) [reply docs/reference/tools/pd-control.md]: It's a valid suggestion - the PD control reference is not very friendly to read and update as well. We can work something out after 3.0 release.

File: docs/reference/tools/pd-control.md (hunk 2 above)
Comment (dcalvin) [reply docs/reference/tools/pd-control.md]: PTAL @Hoverbear.

File: docs/reference/tools/pd-control.md [hunk 3]
```diff
@@ -98,48 +98,119 @@ Use this command to view or modify the configuration information.
...
+    "address": "",
+    "interval": "15s"
+  },
+  "schedule": {
+    "max-snapshot-count": 3,
+    "max-pending-peer-count": 16,
+    "max-merge-region-size": 20,
+    "max-merge-region-keys": 200000,
+    "split-merge-interval": "1h0m0s",
+    "enable-two-way-merge": false,
```
Comment (nolouch) [inline docs/reference/tools/pd-control.md]:
```suggestion
    "enable-one-way-merge": false,
```

Comment (dcalvin) [reply docs/reference/tools/pd-control.md]: Should these two be included in the PD-ctl command description?

**Comment by dcalvin** [comment]
@Hoverbear @nolouch please approve if there is no further issue.

**Pull request merged**